### PR TITLE
traverse to the top-most parent on JSON

### PR DESCRIPTION
**Pull request by ardaakcabuyuk**
WHAT I DID
------------
Added code to traverse to the top-most parent on JSON

HOW TO TEST
---------------
run `python parser.py`

### CODE REVIEW

File: parser.py [hunk 1]
```diff
@@ -11,11 +11,13 @@
         commits[info["data"]["commit"]] = info
         parents[info["data"]["commit"]] = [commits[parent] for parent in info["data"]["parents"]]
 
-root = parents["d9f1e643a6d867c8b0350b88e44de0dbb078f445"]
+a = parents["d9f1e643a6d867c8b0350b88e44de0dbb078f445"]
```
Comment (elifozerr) [inline parser.py]: Why did you choose this name? This could be more sensible.

File: parser.py [hunk 2]
```diff
@@ -11,11 +11,13 @@
...
 
 i = 0
 while len(a[0]["data"]["parents"]) > 0:
     id = a[0]["data"]["parents"][0]
     a = parents[id]
     print("iteration", i, id)
     i += 1
+
+print(a)
```
Comment (elifozerr) [inline parser.py]: Can you delete this line before merging. Thank you.

**Review by elifozerr — CHANGES_REQUESTED** [review]
Can you change these?

File: parser.py (hunk 1 above)
Comment (ardaakcabuyuk) [reply parser.py]: I will change this to `parent = parents["d9f1e643a6d867c8b0350b88e44de0dbb078f445"]`

**Review by elifozerr — APPROVED** [review]
Thanks, you can merge the code now.

**Pull request merged**